source/Tests/IntegrationTests.AndThenThePackageContentsShouldBe.approved.txt

```diff
@@ -1,94 +1,159 @@
 0008b334bed7f0f62057fcd0ead4e42a\NuGet.Frameworks.dll
+005448e07a2776d692f828d68a9480a2\System.ComponentModel.DataAnnotations.dll

```

Comment:
All these extra files are added because `Calamari` is built with `netcoreapp2.1` and `Calamari.Terraform` with `netcoreapp3.1`.